src/kemCombiner_MLKEM768_ECDH_P256_HMAC-SHA256.md

```diff
@@ -2,33 +2,36 @@ Example of id-MLKEM768-ECDH-P256-HMAC-SHA256 Combiner function output.
 
 # Inputs
 mlkemSS:
-2d1c1b18fe729cce3b7cbef3b66b64ef7ecab84bdfb0001d6a7ccc261ae60c52
+203e6728c8ed19267ef1d384856906a645b3f3b5b4e0dc69334072e3a30dd562
 
 tradSS:
-6c01bc04f4f685817144f0506305020ed24dc435a2b41eab2abe074a2102e7ac
+c72614a4a5aeb8eeb7f68fac5305b094b57a86ef4de916b9aa538bc03920ebff
 
-tradCT:  04085bbe80acaf50c3b4c4879f742318bc2a959dd740c516c7f50ea6f880e
-7894f2505cc1c8c75612c9fe647e5bc66edad4c0d30ca697ba25a44f166c3edc1461d
+tradCT:  0490b66eb28ba121aac1f4e2558153bb1ca095cce5d38e49c690c581a8
+e008c98eb7e334ab93518a8cc88537ded4f8154a247fee41e99d0c179b36459bfe6
+c8d3c
 
-tradPK:  04473f58657fce36dd143707bca9c0a9502f3fe014d6a8bec2f6b9b0fefba
-9f40ec0af45fdafa669af35e865914ef390372d920edb0b2645b7e083be8d978f04ef
+tradPK:  043f3ae2c15fdb1df097d5fcc141398e6e1a945ab167f9eb2f44f2fc2c
+a7085804f38aeb8cd6d20ece09d5e6b1812ed2c7c261d4fd43eff877a7e596040b3
+08bdb
 
 Label:  QSF-MLKEM768-P256-HMACSHA256
 
 
 # Combined KDF Input:
 #  mlkemSS || tradSS || tradCT || tradPK || Label
 
-Combined KDF Input: 2d1c1b18fe729cce3b7cbef3b66b64ef7ecab84bdfb0001d6a
-7ccc261ae60c526c01bc04f4f685817144f0506305020ed24dc435a2b41eab2abe074a
-2102e7ac04085bbe80acaf50c3b4c4879f742318bc2a959dd740c516c7f50ea6f880e7
-894f2505cc1c8c75612c9fe647e5bc66edad4c0d30ca697ba25a44f166c3edc1461d04
-473f58657fce36dd143707bca9c0a9502f3fe014d6a8bec2f6b9b0fefba9f40ec0af45
-fdafa669af35e865914ef390372d920edb0b2645b7e083be8d978f04efQSF-
-MLKEM768-P256-HMACSHA256
+Combined KDF Input: 203e6728c8ed19267ef1d384856906a645b3f3b5b4e0dc6
+9334072e3a30dd562c72614a4a5aeb8eeb7f68fac5305b094b57a86ef4de916b9aa
+538bc03920ebff0490b66eb28ba121aac1f4e2558153bb1ca095cce5d38e49c690c
+581a8e008c98eb7e334ab93518a8cc88537ded4f8154a247fee41e99d0c179b3645
+9bfe6c8d3c043f3ae2c15fdb1df097d5fcc141398e6e1a945ab167f9eb2f44f2fc2
+ca7085804f38aeb8cd6d20ece09d5e6b1812ed2c7c261d4fd43eff877a7e596040b
+308bdbQSF-MLKEM768-P256-HMACSHA256
```

Comment:
Similar to the X-wing label, the "QSF-MLKEM768-P256-HMACSHA256" portion of this should be in hexadecimal (presumably there's a way to fix this globally, just commenting here so it's easy to verify later)

Reply:
Fixed.